cypress/e2e/models/migration/reports-tab/reports-tab.ts

```diff
@@ -71,6 +71,11 @@ export class Reports {
             });
     }
 
+    public static getRiskAppNumber(riskDonutSelector: string): Cypress.Chainable<string> {
```

Comment:
Isn't this method same as getRiskValue() method ? Please check if you can use that method .

Reply:
getRiskValue() asserts/validates that a given number is the same as the one on UI that's why it accepts 2 params but in my case i had to create a method that returns the number on UI

Reply:
Ok Can you pls rename that method to verifyRiskValue() and your method to getRiskValue().

Reply:
yes sure on it

Reply:
done